vcfa/config_test.go

```diff
@@ -847,6 +1113,29 @@ func timeStamp() string {
 //     contains a pattern that matches the test name.
 //  6. If the flag -vcfa-re-run-failed is true, it will only run the tests that failed in the previous run
 func preTestChecks(t *testing.T) {
+	// Run priority tests that are responsible for testing core components before they are
+	// precreated for sharing between other tests:
+	// * vCenter
+	// * NSX Manager
+	if !vcfaSkipPriorityTests {
+		runPriorityTestsOnce(t)
+	}
+
+	// Prevent executing a test twice in case it was already executed in
```

Comment:
Missing end of sentence?

Reply:
I think it doesn't even need it. Removed.